addons/t9n/views/t9n_project_views.xml

```diff
@@ -22,7 +22,7 @@
                         <page string="Description">
                             <group>
                                 <group>
-                                    <field name="src_lang_id" domain="[('id', 'not in', target_lang_ids)]"/>
+                                    <field name="src_lang_id" domain="[('id', 'not in', target_lang_ids)]" options="{'no_create': True}"/>
```

Comment:
I'd leave the ability to create a new language at least to the admin. But that's ok for now